packages/language-server/test/plugins/typescript/testfiles/completions/importcompletions8.svelte

```diff
@@ -0,0 +1,7 @@
+<script context="module">
```

Comment:
I added a `importCompletion8.svelte` file in last PR. Maybe we can rename it to `importcompletions8` for consistency. And rename this to `importcompletions9` to clear confusion

Reply:
Oh didn't see that somehow. Your suggestion makes sense